test/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ ExternalProject_Add(
     GIT_REPOSITORY "https://github.com/google/googletest.git"
     SOURCE_DIR "${CMAKE_SOURCE_DIR}/../vendor/googletest"
```

Comment:
Well that's not recommended anyway, that's why we use a build system so that you don't have to "configure" (in our case run cmake) every time, only make.
The test include in our case is part of the test build so deleting your build folder should remove the test include, i.e. it should revert to the situation before doing the build.

In any case we do out-of-source builds which means building should never touch the SOURCE_DIR so the folder has got to go.